specs/sharding/beacon-chain.md

```diff
@@ -718,9 +718,10 @@ def charge_confirmed_header_fees(state: BeaconState) -> None:
         get_active_shard_count(state, get_current_epoch(state))
         * SLOTS_PER_EPOCH * GASPRICE_ADJUSTMENT_COEFFICIENT
     )
-    previous_epoch_start_slot = compute_start_slot_at_epoch(get_previous_epoch(state))
+    previous_epoch = get_previous_epoch(state)
+    previous_epoch_start_slot = compute_start_slot_at_epoch(previous_epoch)
```

Comment:
Good fix, but this reminds me we should limit some of these methods to epoch 1+, epoch 0 doesn't work quite as expected. The previous epoch var would be clipped to 0, and then it tries to confirm current-epoch shard headers.